.github/workflows/terraform.yml

```diff
@@ -48,4 +48,20 @@ jobs:
         uses: aquasecurity/tfsec-pr-commenter-action@v1.0.2
         with:
           tfsec_args: --concise-output
-          github_token: ${{ github.token }}
\ No newline at end of file
+          github_token: ${{ github.token }}
+
+  checkov:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+      - name: Run Checkov
+        uses: bridgecrewio/checkov-action@v12.1964.0 # checkov-action v12.1964.0 == checkov 2.2.182 - keep pre-commit and github action in sync
```

Comment:
I would prefer not to pin to a specific version here.

We had problems in the past using tfsec, where people pinned a version and then when we moved to a newer version had pages and pages of issues to fix. I'd rather we pin to a major version or use latest and fix issues as they come up instead of create a big backlog for the future.

Reply:
will chat with you offline